.github/workflows/bump.yml

```diff
@@ -0,0 +1,29 @@
+name: Bump package version in dependent repos
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  bump:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+      - name: Get version from package.json before release step
+        id: extractver
+        run: echo "::set-output name=version::$(npm run get-version --silent)"
+      - name: Get name of package from package.json
+        id: extractname
+        run: echo "::set-output name=packname::$(npm run get-name --silent)"
+      - name: Bumping latest version of this package in other repositories
+        uses: derberg/org-projects-dependency-manager@v1
+        with:
+          github_token: ${{ secrets.GH_TOKEN }}
+          committer_username: asyncapi-bot
+          committer_email: info@asyncapi.io
+          #This is commit message and PR title for repos where this package is in dependencies
+          commit_message_prod: 'fix: update ${{ steps.extractname.outputs.packname }} to ${{ steps.extractver.outputs.version }} version'
```

Comment:
Does it really matter if it's in the dependencies or not? I mean, if I see `fix` I immediately think it's a bug fix.

Reply:
if it is dev dependency it should not trigger release. For prod dependency it is hard to say, but minimum should be patch in my opinion, so must be `fix` to trigger release in dependent project

Reply:
You're right 👍